tools/monorepo-merge/src/commands/transfer-issues/index.ts

```diff
@@ -164,9 +165,11 @@ export default class TransferIssues extends Command {
 
 		try {
 			await authenticatedGraphQL( '{ viewer { id } }' );
-		} catch ( err: any ) {
-			if ( err?.status === 401 ) {
-				this.error( 'The given token is invalid' );
+		} catch ( err: unknown ) {
+			if ( err instanceof RequestError ) {
+				if ( err?.status === 401 ) {
```

Comment:
I'm getting a complaint here.

<img width="636" alt="image" src="https://user-images.githubusercontent.com/1922453/191661766-35fc8b5d-621a-4a1d-9d9a-8af6f29aa81d.png">

Which is odd because it feels like `RequestError` is just an empty object type.

Reply:
@psealock you need to do a `pnpm i` probably. this is a new dependency of this module

Reply:
Thats the fix, thank you